docs/operating-eck/eck-permissions.asciidoc

```diff
@@ -0,0 +1,112 @@
+:page_id: eck-permissions
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+
+[id="{p}-{page_id}"]
+= ECK permissions
```

Comment:
Should we be more explicit in the title? Something like: "Required RBAC permissions"